nncf/config/structure.py

```diff
@@ -11,7 +11,43 @@
  limitations under the License.
 """
 
+from typing import Optional
+
+from nncf.common.initialization import NNCFDataLoader
+
+
 class NNCFExtraConfigStruct:
     @classmethod
     def get_id(cls) -> str:
         raise NotImplementedError
+
+
+class BNAdaptationInitArgs(NNCFExtraConfigStruct):
```

Comment:
You moved this from torch to, basically, a common folder. Is this object general enough to cover all framework cases?

Reply:
I think so at this stage.